scala-package/examples/src/main/java/org/apache/mxnetexamples/infer/javapi/objectdetector/README.md

```diff
@@ -0,0 +1,116 @@
+# Single Shot Multi Object Detection using Scala Inference API
+
+In this example, you will learn how to use Scala Inference API to run Inference on pre-trained Single Shot Multi Object Detection (SSD) MXNet model.
```

Comment:
Has someone followed this README and verified that it works for Java. I still see paths referring to Scala folders?
@piyushghai @zachgk @lanking520 could one of you try to follow step by step as a naive user and verify this README works for Java

Reply:
There are still some references to the word Scala but the commands have all been updated and should work. Someone else running through the steps would definitely be welcome.

I've got a task today to start updating documentation for the Java API and I plan to include this as well.

Reply:
I'll also be running through these steps since I also need to work on portions of the documentation

Reply:
offline conversation with Andrew - he will do a cleanup along with the other docs he is writing for Java API